src/zenml/zen_stores/migrations/versions/5994f9ad0489_introduce_role_permissions.py

```diff
@@ -50,24 +50,24 @@ def upgrade() -> None:
     conn.execute(
         sa.text(
             """
-            DELETE FROM userroleassignmentschema
-            WHERE role_id
-            IN (SELECT id FROM roleschema WHERE name=='admin' OR name=='guest')
+            DELETE U FROM userroleassignmentschema U
+            LEFT JOIN roleschema R ON U.role_id = R.id
```

Comment:
Why do you need a left join here? `role_id` should never be `null`, and even if it were, those entries would be filtered by the WHERE clause later, so this seems unnecessary.

```suggestion
            JOIN roleschema R ON U.role_id = R.id
```